resource-service/README.md

```diff
@@ -50,9 +50,14 @@ The service accepts a `ResourceRequest` from the User Service, finds all the res
 
 ## Kafka Interface
 
-The application receives 3 messages for each token, a `START` message, a message containing a `ResourceRequest` and an `END` message. The `START` gets consumed by the service, it is then acknowledged as the start of the resources and is then written to the `resource` Kafka topic.
-
-The `ResourceRequest` message then gets consumed by the service and for each resource a `ResourceResponse` object is created. This then gets written to the `resource` Kafka topic. Once all the`ResourceResponse`s have been written to the topic, the `END` message gets written to the `resource` topic to mark the end of the resources for this request. If any errors are thrown within the service, the original request, along with the thrown exception are captured in an `AuditErrorMessage` and written to the Kafka `error` topic.
+The application receives 3 messages for each token.
+A `START` message, a message containing a `ResourceRequest` and an `END` message.
+In order to determine which message is the `START` and which message is the `END` these values are written into the `x-stream-marker` header for each of the received message.
```

Comment:
I'd either rephrase this (the start and end are found in the `x-stream-marker` header) or mention that these values come from the Palisade Service

Reply:
Updated